src/main/java/io/stargate/sgv2/jsonapi/exception/JsonApiException.java

```diff
@@ -59,7 +59,7 @@ public CommandResult get() {
   public CommandResult.Error getCommandResultError(String message) {
     Map<String, Object> fields =
         Map.of("errorCode", errorCode.name(), "exceptionClass", this.getClass().getSimpleName());
-    return new CommandResult.Error(message, fields);
+    return new CommandResult.Error(message, fields, 200);
```

Comment:
fixed? so all `JsonApiExceptions` would return 200 always? would add a method into `ErrorCode.getHttpStatusCode()` which can return `200` by default.. then we can easily change for some error code..

Reply:
Yes any error other than specific one in the ticket will return error message with http status code 200 as per #407

Reply:
OK.